src/plugins/plugin.title.js

```diff
@@ -230,15 +230,15 @@ export default {
 	_element: Title,
 
 	beforeInit(chart) {
-		const titleOpts = chart.options.title;
+		const titleOpts = chart.options.plugins.title;
```

Comment:
I thought the plugin options get passed in?

Reply:
They get, but are not used in core plugins. I've tried once changing that some weeks ago, but hit some obstacles. I'd think it was (at least) those tests that are disabled in this PR.